foundations/src/telemetry/settings/tracing.rs

```diff
@@ -125,3 +113,56 @@ impl JaegerThriftUdpOutputSettings {
         server_addr
     }
 }
+
+/// Settings used when active sampling is enabled.
+#[cfg_attr(feature = "settings", settings(crate_path = "crate"))]
+#[cfg_attr(not(feature = "settings"), derive(Clone, Debug, serde::Deserialize))]
+pub struct ActiveSamplingSettings {
+    /// Sampling ratio.
+    ///
+    /// This can be any fractional value between `0.0` and `1.0`.
+    /// Where `1.0` means "sample everything", and `0.0` means "don't sample anything".
+    #[serde(default = "ActiveSamplingSettings::default_sampling_ratio")]
+    pub sampling_ratio: f64,
+
+    /// Settings for rate limiting emission of traces
+    pub rate_limit: RateLimitingSettings,
+}
+
+impl ActiveSamplingSettings {
+    fn default_sampling_ratio() -> f64 {
+        1.0
+    }
+}
+
+#[cfg(not(feature = "settings"))]
+impl Default for ActiveSamplingSettings {
+    fn default() -> Self {
+        Self {
+            sampling_ratio: ActiveSamplingSettings::default_sampling_ratio(),
+            rate_limit: Default::default(),
+        }
+    }
+}
+
+/// The sampling strategy used for tracing purposes.
+#[cfg_attr(
+    feature = "settings",
+    settings(crate_path = "crate", impl_default = false)
+)]
+#[cfg_attr(not(feature = "settings"), derive(Clone, Debug, serde::Deserialize))]
+pub enum SamplingStrategy {
+    /// This only samples traces which have one or more references.
+    ///
+    /// Backed by [cf_rustracing::sampler::PassiveSampler]
+    Passive,
```

Comment:
Could we elaborate here a bit more about active vs passive sampling in doc comments for these options?